.changeset/fresh-chairs-wait.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/common-utils": major
+---
+
+EventForwarder removed from common-utils
+
+The EventForwarder class in common-utils was deprecated and has now been removed.
```

Comment:
Is there any migration guidance we should offer here for affected users?

Reply:
If not, we should probably note that explicitly.

Reply:
Yeah it's a bad pattern I'm trying to remove so no replacement recommended.  I'll update the changeset.